aws-member-account-iam-roles.yaml

```diff
@@ -55,9 +31,9 @@ Resources:
                 - !Ref MasterAccountId
             Action:
               - sts:AssumeRole
-            Condition:
-              Bool:
-                aws:MultiFactorAuthPresent: 'true'
+            # Condition:
+            #   Bool:
+            #     aws:MultiFactorAuthPresent: 'true'
```

Comment:
@maybealexandra can you please delete these in a separate commit vs commenting them out?

Reply:
done